langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiChatModel.java

```diff
@@ -112,7 +112,7 @@ public OpenAiChatModel(String baseUrl,
         this.seed = seed;
         this.user = user;
         this.maxRetries = getOrDefault(maxRetries, 3);
-        this.tokenizer = getOrDefault(tokenizer, OpenAiTokenizer::new);
+        this.tokenizer = OpenAiTokenizer.getTokenizerOrDefault(tokenizer, this.modelName);
```

Comment:
```suggestion
        this.tokenizer = getOrDefault(tokenizer, () -> OpenAiTokenizer.from(this.modelName));
```

What about keeping `getOrDefault` logic here?